components/brave_rewards/resources/page/reducers/rewards_reducer.ts

```diff
@@ -329,6 +329,7 @@ const rewardsReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State
       const { value, error } = action.payload.result
 
       if (value) {
+        chrome.send('brave_rewards.getUserType')
```

Comment:
Could you just leave a brief comment in the PR description on why this line got added here?

Reply:
Will do, thanks.